docs/creating_review_app.md

```diff
@@ -38,70 +17,65 @@ CircleCI will match the `review-app-` prefix and either:
 
-> Once you're done working with the review app it (currently) has to be deleted manually
-
-#### Deleting a Review App
-
-Run:
-
-```sh
-yarn delete-review-app <name>
-```
-
-The `name` is the name of the branch, minus the `review-app-` prefix.
-
-#### Manual Steps
-
-##### Manual Build
+### Building on Local
 
 First, make sure `jq` is installed:
 
 ```sh
 brew install jq
 ```
 
-Then launch the script:
+Then launch the script with:
 
 ```sh
-./scripts/build_review_app.sh review-app-name
+./scripts/build_review_app.sh awesome-feature
 ```
 
-##### Manual Update
+The script will save a K8s spec in `hokusai/awesome-feature.yml`
 
-If you want to push subsequent changes to the review app you can push a new build to the same tag with the `--overwrite` flag:
+### Accessing the review app.
 
-```sh
-hokusai registry push --overwrite --skip-latest --force --tag <name>
-```
+To access the review app, you must create a DNS name for it. A name ending in `artsy.net` is required for full OAuth flows (for, say, login redirects between Gravity and Force for Auction registration). On Cloudflare, please do:
 
-and you need to redeploy your app:
+1. [Login to Cloudflare](https://dash.cloudflare.com/), and navigate to **artsy.net** > **DNS**
+1. Click `+ Add Record`
+1. Change `Type` dropdown to `CNAME`
+1. Under `Name` enter `awesome-feature`
+1. Under `Target` say `nginx-staging.artsy.net`
+1. Hit `Save`
+1. DNS will propagate and after a few minutes the review app will be available via `awesome-feature.artsy.net`
+
+### Updating the review app
+
+If you want to push subsequent changes to the review app, push a new build to the same tag with the `--overwrite` flag:
 
 ```sh
-hokusai review_app deploy <name> <name>
+hokusai registry push --overwrite --skip-latest --force --tag awesome-feature
 ```
 
-😇 After your review app is no longer needed please remember to clean up any CNAMEs you've created, and to de-provision the review app itself with `hokusai review_app delete <review-app-name>`
+and redeploy the app:
 
-For more info on Review App maintenence, [see Hokusai docs](https://github.com/artsy/hokusai/blob/master/docs/Review_Apps.md).
+```sh
+hokusai review_app deploy awesome-feature awesome-feature
+```
 
-#### DNS Setup
+### Deleting a Review App
 
-Regardless of how you created a review app (CircleCI or manually), `build_review_app.sh` will output a hostname like the following:
+When you are done using the review app, please delete it by:
 
 ```sh
-a99199101d01011e9aff2127c3b176f7-1359163722.us-east-1.elb.amazonaws.com
+yarn delete-review-app awesome-feature
 ```
 
-This is your Review App url, which should support all of the basic features inside of Force.
-
-If you'd like a pretty URL subdomain or need to test full OAuth flows (for, say, login redirects between Gravity and Force for Auction registration) then an additional non-automated step is required via Cloudflare:
+And please delete its DNS entry by:
```

Comment:
is "please" to nice? should we be more assertive ... `You should delete review apps as soon as QA is complete.`

Reply:
Works for me. Updated.